src/plugins/wb-delete-pii/wb-delete-pii-en.hbs

```diff
@@ -0,0 +1,125 @@
+---
+{
+	"title": "Delete-PII",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Delete Personal Identifiable Information (PII) before form submit",
+	"tag": "wb-delete-pii",
+	"parentdir": "wb-delete-pii",
+	"altLangPrefix": "wb-delete-pii",
+	"dateModified": "2022-02-22"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+<div class="alert alert-warning">
+	<h2>Limited feature</h2>
+	<p>Compatible with regular forms or with forms using Postback plugin. This feature will <strong>not work</strong> with <a href="https://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-en.html">Form validation</a> plugin. </p>
+</div>
+
+<h2>Example using regular form</h2>
+
+<p>Delete Personal Identifiable Information (PII) from a form fields before submit.</p>
+
+<form action="wb-delete-pii-en.html" method="GET" class="wb-delete-pii" data-wb-delete-pii="true">
+	<div class="form-content">
+		<div class="form-group">
+			<label for="firstname"><span class="field-name">First Name</span></label>
+			<input class="form-control" id="firstname" name="firstname" type="text" data-rule-minlength="2" />
```

Comment:
No need for Form validation data attributes in the example since we are not using it (**SAME FOR FRENCH**)
```suggestion
			<input class="form-control" id="firstname" name="firstname" type="text" />
```